src/parser/whamm.pest

```diff
@@ -63,8 +63,8 @@ fn_call = { ID ~ "(" ~ ( arg )? ~ ( "," ~ arg )* ~ ")" }
 // var ops
 declaration = { TYPE ~ ID }
 assignment = { ID ~ "=" ~ (ternary | expr) }
-statement = { ( ( declaration | assignment | fn_call ) ~ ";" )+ }
-
+statement = { ( ( declaration | assignment | fn_call | incrementor ) ~ ";" )+ }
```

Comment:
I will recommend not changing the definition of statements, but change `expr` instead.  Define `++` as a postfix. See https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-924.pdf page 24.

Reply:
Original branch of work had it like that, but it becomes a nightmare in the emitter. This solution disallows some behavior, but ends up much less complex